src/main/java/address/updater/UpdateProgressNotifier.java

```diff
@@ -0,0 +1,55 @@
+package address.updater;
```

Comment:
On second thought, this should be moved to commons since it is used by 2 components. Otherwise, it will be difficult to track which components need to be updated (since their source code are placed everywhere).